src/server/api/routes/studentsprofile.router.js

```diff
@@ -0,0 +1,62 @@
+/* The API routing for events */
+
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const studentsProfileController = require('../controllers/studentsprofile.controller');
+
+/**
+ * @swagger
+ * /students/profile:
+ *  get:
+ *    summary: Get all students profile
+ *    description:
+ *      Will return all students profile.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+
+router.get('/', (req, res, next) => {
+  studentsProfileController
+    .getStudentsProfile()
```

Comment:
You need to pass id here
```suggestion
    .getStudentsProfile(req.user.id)
```

Reply:
Yuka, I think this suggestion also need to be addressed( getStudentsProfile(req.user.id))